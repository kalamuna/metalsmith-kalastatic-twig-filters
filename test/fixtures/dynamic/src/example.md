---
view: layout.html
title: Hello World!
---
