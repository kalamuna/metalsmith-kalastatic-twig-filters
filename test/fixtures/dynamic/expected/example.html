<html>
<body>

  <h1>Encrypted: Ncjji&amp;Qitjb&#039;</h1>

</body>
</html>
